core/environment/utils.go

```diff
@@ -28,38 +28,47 @@ import (
 	"bytes"
 	"encoding/json"
 	"fmt"
-	"github.com/AliceO2Group/Control/core/the"
-	"gopkg.in/yaml.v3"
 	"os"
 	"sort"
+
+	"github.com/AliceO2Group/Control/core/the"
+	"gopkg.in/yaml.v3"
 )
 
-func parseWorkflowPublicInfo(workflowExpr string) (bool, string, error) {
+type WorkflowPublicInfo struct {
```

Comment:
```suggestion
type WorkflowInfo struct {
```
the name could be updated.

Reply:
ah ok, i think i understand what "public" refers to now.